src/azul/plugins/metadata/hca/transform.py

```diff
@@ -1360,6 +1373,7 @@ def _entities(self) -> Iterable[api.File]:
 
     def _transform(self, files: Iterable[api.File]) -> Iterable[Contribution]:
         zarr_stores: Mapping[str, List[api.File]] = self.group_zarrs(files)
+        aggregate_dates = self._aggregate_dates(set(), set())
```

Comment:
This is a fairly roundabout way of getting to the default values. Since aggregate dates don't apply to non-container entities, the aggregate date stuff should sort of disappear rather than require this boilerplate in each transformer. I'll sketch something out.